configs/manager/halfstack.toml

```diff
@@ -30,6 +30,8 @@ disabled-plugins = []
 
 hide-agents = true
 
+max-session-hangtime = { "PREPARING" = "1h", "TERMINATING" = "30m" }
+
```

Comment:
Characteristics of this config is more suitable with Etcd (shared) config, rather than `manager.toml`. Let's refactor the code so that we can find this config at the shared config.